integration-tests/jpa-repository/src/main/java/io/quarkiverse/jberet/it/jdbc/JpaBatchlet.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkiverse.jberet.it.jdbc;
```

Comment:
Shouldn't this be in the `jpa` package?

Reply:
ops, yea you're so right
I just fixed that